addons/web/static/src/views/fields/copy_clipboard/copy_clipboard_field.js

```diff
@@ -32,7 +32,7 @@ CopyClipboardField.props = {
 };
 CopyClipboardField.extractProps = ({ attrs }) => {
     return {
-        label: attrs.options.label,
+        label: attrs.string,
```

Comment:
```suggestion
        string: attrs.string,
```
Can't you simply rename the whole `label` thing to `string` :thinking:

Reply:
Yes, better that way